pdtable/io/parsers/fixer.py

```diff
@@ -31,6 +31,7 @@ def __init__(self):
         self.table_name = None
         self.column_name = None
         self.table_row = None
+        self.parameters = dict()
```

Comment:
Instead of using a dictionary, please use a boolean instead. It makes for more explicit code
```suggestion
        self.strict_types = True
```

Reply:
Make sense, done.